package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@skyux/angular-tree-component",
+  "version": "1.0.0-alpha",
+  "description": "SKYUX styles and utilities to be used with angular-tree-view",
+  "main": "bundles/bundle.umd.js",
+  "module": "index.ts",
+  "scripts": {
+    "build": "skyux build-public-library",
+    "skyux": "skyux",
+    "start": "skyux serve --hmr",
+    "test": "skyux test --coverage library --logFormat none",
+    "test:visual": "skyux e2e --logFormat none"
+  },
+  "keywords": [],
+  "author": "Blackbaud",
+  "license": "ISC",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/blackbaud/skyux-angular-tree-component"
+  },
+  "peerDependencies": {
+    "@angular/common": "^7.0.0",
+    "@angular/core": "^7.0.0",
+    "@skyux/core": "^3.7.0",
```

Comment:
Do these peers actually need to be this high? I know we would rather have people upgrading consistently; however, we also shouldn't force that if it isn't a true requirement.

Reply:
Yep, they need core `3.7.0` for the `getFocusableChildren()` method that helps with keyboard nav.